bin/create_tier2_ugent_home_data_directory_tree.py

```diff
@@ -66,7 +66,7 @@ def main():
 
     storage_settings = VscStorage()
 
-    local_storage_conf = configparser.SafeConfigParser()
+    local_storage_conf = configparser.ConfigParser()
```

Comment:
why is this needed?

Reply:
safeconfigparser was renamed to configparser in python3. the old "unsafe" configparser is gone, and safeconfigparser is deprecated.